### DISCUSSION TITLE
Conformacion de Grupo

### DISCUSSION THREAD

**Discussion by gonaiot** [ISPC-TST-EM2022/Grupo7#8] [General]
Que tal chicos, queria saber que paso con el integrante que falta en su grupo. 

![image](https://user-images.githubusercontent.com/108846060/183221622-9389809a-6f07-4699-867f-2086495b25ae.png)

**Comment by chulkx** [comment]
Buenas profe, disculpe, no habia visto el msj. Ya lo agregamos al grupo. El tuvo problemas con la plataforma de ispc, y cuando lo solucionó se sumó al grupo.
Por otro lado, otros integrantes del grupo ya no estarán cooperando xq decidieron continuar el próximo año.

**Reply by gonaiot** [reply]
Muy bien, muchas gracias.
